packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -244,11 +244,11 @@ curl -v -X GET \
 
 The submitted Inline Hook properties will replace the existing properties after passing validation. Note that some properties are immutable and cannot be updated. Refer to the description of each property in the [Inline Hook object](#inline-hook-object) table for information.
```

Comment:
no future tense, let's pull that "note" content out and create an actual note for it, and we use contractions for a more conversational tone.

The submitted Inline Hook properties replace the existing properties after passing validation.

> **Note:** Some properties are immutable and can't be updated. Refer to the description of each property in the [Inline Hook object](#inline-hook-object) table for information.